avalon/fusion/pipeline.py

```diff
@@ -4,6 +4,8 @@
 import logging
 from pyblish import api as pyblish
 
+_FORCED_COMP = {"COMP": None}
```

Comment:
With the variable called `_FORCED_COMP` what's the reason for making it a dictionary that holds a comp. Couldn't it just be `_FORCED_COMP = None` and then assign it the comp elsewhere (since it's global in the scope of this module?)

Reply:
It did threw an undeclared variable error in `force_current_comp()`, I fixed it this way.

Reply:
Ah, yes... you are assigning it there - which makes it just assign it to a local variable there. :+1: 

Other way could be this: https://github.com/Colorbleed/core/blob/master/avalon/maya/pipeline.py#L21